### PR TITLE
Webpage is up

### PR DESCRIPTION
**Pull request by thetaxas2468**
Webpage is up !

### CODE REVIEW

**Review by RawanAbuleil — APPROVED** [review]

File: buy_together_app/templates/buy_together_app/main.html [hunk 1]
```diff
@@ -0,0 +1,59 @@
...
+        </div>
+        <div class="bigFont">
+            <p>The idea of the project is to let group of people to organize together and buy straight from the supplier at lower cost.</p>
+            Goals
+            <br>
+            - to create simple and easy communication between an organized group of people and a supplier.
+            <br>
+            - lower the prices of food cost
+            <br>
+			Our plan is to work by the division we have created between us and to make a beatiful Front-End page with decent design that has the ability to communicate with the backend that is connected with the database and after all that done it gives the clients the ablility to buy straight from the suppliers which mean the lower the prices of the food will cost and also with easy communication betwen clients and suppliers.        </div>
```
Comment (pazMenachem) [inline buy_together_app/templates/buy_together_app/main.html:57]:
```suggestion
Our plan.
In order to execute our idea, we intend to build a web application, we will divide our 
work between us(five members total) and each will be responsible for each section of the project.
```

**Review by pazMenachem — COMMENTED** [review]
suggestion for more focused plans

File: buy_together_app/templates/buy_together_app/main.html (hunk 1 above)
Comment (thetaxas2468) [reply buy_together_app/templates/buy_together_app/main.html]: Done

**Comment by pazMenachem** [comment]
LGTM!

**Comment by yftacherzog** [comment]
@thetaxas2468 good job! I know it's been a bumpy road this one. You did well. For next time, please write more descriptive commit message title and body, explaining what the commit is about.

**Pull request merged**